resources/css/filament/admin/theme.css

```
@@ -9,6 +9,7 @@
 
 @source '../../../../app/Filament/**/*';
 @source '../../../../resources/views/filament/**/*';
+@source '../../../../resources/js/admin/routeforge/**/*';
```

Stylelint (17.11.1)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)